lib/action_view/component/base.rb

```diff
@@ -232,7 +232,13 @@ def template_errors
           @template_errors ||=
             begin
               errors = []
-              errors << "#{self} must implement #initialize." if source_location.nil?
+
+              unless const_source_location_supported?
```

Comment:
Can we combine this conditional with `if source_location.nil?`? It's a little confusing why the two are separate.

Reply:
@joelhawksley addressed! But now CI seems to be failing at the bundle step for some reason… 😕 Will try re-running it.